pkg/util/utils.go

```diff
@@ -173,19 +171,8 @@ func FriendlyDuration(d time.Duration) time.Duration {
 func CommandExecutesSuccessfully(cmd string, args ...string) bool {
 	var runner Runner = &RealRunner{}
 	var stdOut, stdErr io.Writer
-	// check in the path if cmd	is present
```

Comment:
As said, no need for all this

Reply:
Right, path lookup is done by _exec.Command_ itself, i was not adding the dummybinaries to the path, instead tried to run "ls" in case of darwin/linux and "dir" in windows. But that returns exitcode 1 all the time, even though those binaries are always in PATH.